README.md

```diff
@@ -75,6 +75,7 @@ The `use-package` configuration below will bind `C-c #` in the `org-mode-map` (w
                                   (org-custom-cookies-update-containing-subtree)))))
   :config
   (add-hook 'org-clock-out-hook 'org-custom-cookies-update-containing-subtree)
+  (add-hook 'org-font-lock-set-keywords-hook 'org-custom-cookies--cookie-face-for-all-custom-cookies)
```

Comment:
I think instead of asking the user to add this manually, I think it would be a better user experience if they set a configuration option like `org-custom-cookies-use-statistics-font`, and we added the hook for them. What do you think?

Reply:
I think that would be better, but how would implement something like that?

Reply:
@gsingh93 Just thought I would ping you again in case you missed this message.

Reply:
We could use the `:set` keyword when defining the variable with `defcustom`, which will call a callback function: https://www.gnu.org/software/emacs/manual/html_node/elisp/Variable-Definitions.html

This would require the user interactively customizing the variable, manually calling `custom-set-variable`, or putting it in the `:customize` section of a `use-package` block. Reading the docs, it seems like this would also still work if the user used `setq` to set the variable before the package is loaded, like in an `:init` block. The only thing that wouldn't work is if it's set with `setq` in a `:config` block, or any other time after the package has been loaded.

I've never done this before, but it seems like a good solution. What do you think?